osu.Game/Overlays/Profile/Sections/PaginatedProfileSubsection.cs

```diff
@@ -102,11 +106,22 @@ private void onUserChanged(ValueChangedEvent<UserProfileData?> e)
 
             if (e.NewValue?.User != null)
             {
-                showMore();
+                pendingRequestOnVisibility = true;
```

Comment:
I'm 100% sure you could make this fail with some tight timings. You're using the default values for DLUW which means the unload only happens after 1 second. If a new user is set within that 1 second and the section is still visible, you can imagine what happens.

You're going to want to use a `DelayedLoadWrapper` and add a new one when the user changes.

Reply:
Should be addressed in https://github.com/ppy/osu/pull/33647/commits/35faee5647e22397d3a941f2ad9aab1b5391cae9